hey @henry-lp,
when trying to build your branch, I discovered that you have some checkstyle violations:
![Screenshot 2020-07-17 at 15 32 48](https://user-images.githubusercontent.com/15121999/87791615-daa71080-c842-11ea-94c5-2d2fd9c84a35.png)

Output is: 

[ERROR] commons/src/main/java/org/eclipse/kapua/commons/jpa/ResourceSqlScriptExecutor.java:[2] (header) RegexpHeader: Line does not match expected header line of '^ \* Copyright \(c\) \d{4}(|, \d{4}) (.*) and others\.?$'.


[ERROR] rest-api/resources/src/test/java/org/eclipse/kapua/app/api/resources/v1/resources/TagsTests.java:[1] (header) RegexpHeader: Line does not match expected header line of '^/\*{79}$'.


[ERROR] rest-api/resources/src/test/java/org/eclipse/kapua/app/api/resources/v1/resources/TagsTests.java:[11,27] (whitespace) MethodParamPad: '(' is preceded with whitespace.


[ERROR] rest-api/resources/src/test/java/org/eclipse/kapua/app/api/resources/v1/resources/TagsTests.java:[14] (whitespace) EmptyLineSeparator: There is more than 1 empty line one after another.

can you please check? Just run `mvn clean install -DskipTests` on your branch.

--- a/commons/src/main/java/org/eclipse/kapua/commons/jpa/ResourceSqlScriptExecutor.java
+++ b/commons/src/main/java/org/eclipse/kapua/commons/jpa/ResourceSqlScriptExecutor.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 Eurotech and/or its affiliates and others
+ * Copyright (c) 2017, 2020 Eurotech and/or its affiliates and others
  *
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
@@ -89,8 +89,8 @@ public class ResourceSqlScriptExecutor {
             StringBuilder sqlStringBuilder = new StringBuilder();
             try {
                 try (
-                        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(qStr);
-                        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream))) {
+                    InputStream inputStream = Thread.currentThread().getContextClassLoader().getResourceAsStream(qStr);
+                    BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(inputStream))) {
 
                     String sqlLine;
                     while ((sqlLine = bufferedReader.readLine()) != null) {

